src/client/tab.rs

```diff
@@ -189,6 +194,7 @@ impl Tab {
         send_app_instructions: SenderWithContext<AppInstruction>,
         max_panes: Option<usize>,
         pane_id: Option<PaneId>,
+        input_mode: InputMode,
```

Comment:
We keep InputMode both on Tab and Screen - do you think we can only keep it in one place? Maybe Screen? And then pass it to each tab when we render? I think it will make things more maintainable, what do you think?

Reply:
Yeah, don't see why not

Reply:
Actually, we have render called inside the tab itself in lots places, so that would introduce maybe a bit more complexity?